packages/layout/src/ChartPanel.ts

```diff
@@ -196,8 +221,45 @@ export class ChartPanel extends Border2 implements IHighlight {
         this._progressBar.enter(domNode, element);
     }
 
+    preUpdateTiny(element) {
+        element.selectAll("div.body").style("visibility", "hidden");
+    }
+    preUpdateSmall(element) {
+        const scale_x = this.size().width / this._resolutions.small.width;
+        const scale_y = this.size().height / this._resolutions.small.height;
```

Comment:
@GordonSmith Here's where I'm calculating the scale based on the min-width (`this._resolutions.small.width`) and actual-width (`this.size().width`)